CONTRIBUTING.md

```diff
@@ -48,6 +50,35 @@ We add this additional rule:
 
 - Do not rely on GitHub issue reference numbers in commit messages, as we have no guarantee the host system and its autolinking will be stable in time. Make sure the context is self-explanatory. If an external reference is given, use its full URL.
 
+## Development
+
+### Documentation
+
+#### CLI
+
+##### docopt
+
+For command-line examples and documentation, we follow the [docopt usage patterns](http://docopt.org) syntax. Quick recap of the main points:
+
+- mandatory arguments are given between `<` and `>`;
+- optional elements are given between `[` and `]`;
+- mutually exclusive elements are given between `(` and `)` and separated by `|`.
+
+```diff
+- npm start $service_id [, $service_id, ...]
++ npm start <service_id>...
```

Comment:
`service_id` is optional, shouldn't it be written like that `[service_id]` instead?

Reply:
I voluntarily gave an example that did not reflect accurately the current CLI but simply gave examples of how to rewrite. I will adjust it to reflect the CLI.